package.json

```diff
@@ -34,7 +34,9 @@
   },
   "homepage": "https://github.com/jshor/datebook",
   "dependencies": {
-    "file-saver": "^2.0.2"
+    "awesome-typescript-loader": "^5.2.1",
+    "file-saver": "^2.0.2",
+    "source-map-loader": "^0.2.4"
```

Comment:
'awesome-typescript-loader` and `source-map-loader` need to be devDependencies, otherwise it will force the consumer to install them

```sh
yarn remove awesome-typescript-loader
yarn remove source-map-loader
yarn add awesome-typescript-loader -D
yarn add source-map-loader -D
```